examples/faces_detection/faces_detection.ipynb

```diff
@@ -0,0 +1,987 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #20.        (toloka.filter.Skill(detection_skill.id) != None)
</pre><p>It is stated in a comment above that users, who performed detection tasks, shouldn't be allowed to perform verification tasks. But if worker performed detection tasks, their detection skill is not None. So we need to revert the inequality to equality operator: </p><pre class="ql-syntax" spellcheck="false">Line #20.        (toloka.filter.Skill(detection_skill.id) == None)
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Toloka/toloka-kit/pull/119/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='33'/>